doc/lua_api.md

```diff
@@ -5716,14 +5716,21 @@ Utilities
       min_jitter = 0.01,         -- minimum packet time jitter
       max_jitter = 0.5,          -- maximum packet time jitter
       avg_jitter = 0.03,         -- average packet time jitter
+
+      -- The version information is provided by the client and may be spoofed or
+      -- inconsistent in engine forks. You must not use this for checking
+      -- feature availability of clients, please use protocol_version whenever
+      -- it is possible.
+      -- Use this ONLY as a last resort!!!
```

Comment:
```suggestion
      -- inconsistent in engine forks. You must not use this for checking
      -- feature availability of clients. Instead, do use the fields
      -- `protocol_version` and `formspec_version` where it matters.
      -- These version fields may only be used for analysis purposes.
```
1. Added an additional hint for `formspec_version`
2. The `[...] last resort!!!` sentence should not be there. Every positive hint to modders will eventually make them think that it is okay to use it for feature checks anyway. I can imagine the following thoughts:
    * "It works for me, so why not do it?"
    * "Just this check surely won't hurt anyone."